### DISCUSSION TITLE
Classroom Bot doesn't load the autograder into new forked repositories

### DISCUSSION THREAD

**Discussion by mcknijak** [community/community#102024] [GitHub Education]
### Hi everyone,

I recently opted in to the new way of distributing assignments but noticed a massive issue. The classroom bot doesn't copy the action config files into student repos when they accept the assignment. What gives? I know this is beta but it seems like making sure core functionality works before moving to this stage would've been prudent. Please let me know if I missed something it setting it up.

(yes, I know you can get around it by manually writing the config file and putting them into the upstream source repository but if that's the expectation why is the GUI tool still available / why is the this not documented anywhere?)

**Comment by hubertle43100** [comment]
Hello `mcknijak`,

**I would suggest you take these potential first few steps in solving your issue:**


`Check Documentation and Settings:`
Review the documentation for the new assignment system to ensure correct setup and configurations.

`Community Support:`
Seek help on the community forum or support channels to get insights from others facing similar issues.

`Submit Bug Report:`
If the problem persists, submit a concise bug report to the development team with specific details and steps to reproduce the issue.
